build/Dockerfile-setup

```diff
@@ -9,7 +9,26 @@ RUN apt-get update; DEBIAN_FRONTEND=noninteractive apt-get install -y --no-insta
     postgresql-client \
     python \
     sqlite3 \
-    tar
+    tar \
+    apt-transport-https \
+    locales
+
+RUN echo "deb http://deb.debian.org/debian stretch main" > /etc/apt/sources.list.d/stretch.list && \
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
+    curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list | tee /etc/apt/sources.list.d/msprod.list && \
+    apt-get update && ACCEPT_EULA=Y apt-get install -y \
+    msodbcsql=13.1.4.0-1 \
+    mssql-tools \
+    unixodbc-dev
+
+RUN dpkg-reconfigure -f noninteractive tzdata && \
+    sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+    sed -i -e 's/# nb_NO.UTF-8 UTF-8/nb_NO.UTF-8 UTF-8/' /etc/locale.gen && \
```

Comment:
No need in nb_NO.UTF-8 locale and tzdata reconfiguration (see http://stackoverflow.com/a/38553499/746347)